packages/components/src/Card/src/Card.module.css

```diff
@@ -0,0 +1,24 @@
+.hop-Card {
+    /* Internal Variables */
```

Comment:
and if you do end up having internal variables, they're usually below the component variables.